src/com/unbounce/dogstatsd/core.clj

```diff
@@ -32,46 +33,58 @@
                                tags)))))
 
 (defn increment
-  [metric & {:keys [tags sample-rate]}]
-  (let [tags (str-array tags)]
-    (if sample-rate
-      (.incrementCounter client metric sample-rate tags)
-      (.incrementCounter client metric tags))))
+  ([metric]
+   (.incrementCounter client metric -empty-array))
```

Comment:
FWIW i split it up this way because I think this would be a faster path on the critical path because it does away w/ adding sample rate or tags (by calling .incrementCounter directly)